app/controllers/legacy_tasks_controller.rb

```diff
@@ -35,6 +33,34 @@ def index
     end
   end
 
+  def invalid_parameters?
+    return redirect_using_valid_css_id if non_normalized_css_id?(params[:user_id])
+
+    fail(Caseflow::Error::InvalidUserId, user_id: params[:user_id]) unless user
+
+    return invalid_role_error unless ROLES.include?(user_role)
+
+    nil
+  end
+
+  def user_role
+    (params[:role] || user.vacols_roles.first).try(:downcase)
+  end
+
+  def redirect_using_valid_css_id
```

Comment:
Last normalized, I promise!
```suggestion
  def redirect_using_normalized_css_id
```